packages/step-storybook/package.json

```diff
@@ -25,6 +25,7 @@
 	},
 	"peerDependencies": {
 		"@storybook/react": ">= 6",
+		"core-js": ">= 3",
```

Comment:
I don't think we need core-js as a peer since this isn't built with the essex build pipeline